right/src/keyboard_layout.h

```diff
@@ -33,6 +33,32 @@ typedef enum {
     UHK_KEY_LPRESSLAYER,
 } uhk_key_type_t;
 
+enum {
```

Comment:
A short update: with `buttonActions` as a bitfield, adding buttons to the report is as simple as `report->buttons |= key.mouse.buttonActions`. Though, even if it was discrete values, the values would still be the same, so that the code can be this trivial too.

If I turn the buttons into discrete values, a simple enum instead of bit positions, then a button press would be `report->buttons |= (1 << key.mouse.button)`. Still simple, but `|= key.mouse.button` is even simpler.